CHANGELOG.rst

```diff
@@ -15,6 +15,11 @@ Change Log
 
 Unreleased
 
+[3.8.29] 2020-10-06
```

Comment:
_sigh_ so now we're being inconsistent with the dates (not your fault, there's a few examples here)
I know Brandon led the way in getting rid of them, but we seem to be inconsistently adding them back in.
Let's remove it, to try to solidify the trend of removing them :)

Reply:
Mixed feelings, but meh- want to be consistent regardless